tests/yast2_gui/yast2_bootloader.pm

```diff
@@ -53,9 +55,11 @@ sub run() {
     type_string 'dummy-password';
     send_key 'alt-y';
     type_string 'dummy-password';
+    wait_still_screen 3;
```

Comment:
why do you need to wait still screen after typing string? (and why only here - not also after first password?)